res/css/views/messages/_MImageBody.scss

```diff
@@ -54,18 +54,30 @@ $timeline-image-border-radius: 8px;
     }
 }
 
-.mx_MImageBody_thumbnail_container {
-    border-radius: $timeline-image-border-radius;
-
-    // Necessary for the border radius to apply correctly to the placeholder
-    overflow: hidden;
-    contain: paint;
-
-    min-height: $font-44px;
-    min-width: $font-44px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
+.mx_MImageBody {
+    .mx_MImageBody_thumbnail_container {
+        border-radius: $timeline-image-border-radius;
+
+        // Necessary for the border radius to apply correctly to the placeholder
+        overflow: hidden;
+        contain: paint;
+
+        min-height: $font-44px;
+        min-width: $font-44px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        // Override inline height and width values to avoid overflow
+        max-height: 100% !important;
+        max-width: 100% !important;
```

Comment:
I am having trouble seeing why this has to be `!important` - what selector are we overriding?

Reply:
As the comment says, it overrides inline styles, which can basically only be overridden with `!important`, according to https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#complete_cascade_order. 

However, I'm removing the max-height declaration since it is irrelevant here and forcing it seems to break canvas on the placeholder for an image both on bubble message layout and modern layout, like below.

![Untitled](https://user-images.githubusercontent.com/3362943/169655925-fb1c5a28-e686-4df5-8bd7-685a84d660ef.png)

Reply:
Ah! I somehow missed that, sorry

Reply:
Following the commit history, I am not really sure why `max-width` and `max-height`, instead of `width` and `height`, have been used here. Might be related with spinner, according to the line here: https://github.com/matrix-org/matrix-react-sdk/commit/4b5ca1d7a9bd4a369c51f81e6cd5ea4ca50a563e#diff-265bce8083cb791022167e130a59ebba99e3e974d9d3120f953ed41461b0f22aR466